jsonschema2pojo-maven-plugin/src/main/java/org/jsonschema2pojo/maven/Jsonschema2PojoMojo.java

```diff
@@ -445,6 +445,15 @@
     private String targetVersion = "1.6";
 
     /**
+     * Whether to include dynamic getters/setters/builders or to omit these accessor methods.
```

Comment:
This should be `Whether to include dynamic getters, setters, and builders or to omit these methods.` in all documentation.